tissue_bridges/README.md

```diff
@@ -0,0 +1,86 @@
+# Tissue bridges
+
+This folder contains scripts to compare tissue bridges obtained using different methods:
+
+1. **_manual_**: manual measurement of tissue bridges on manually segmented (GT) intramedullary lesions -- values provided by collaborators
+2. **_semi-automatic_**: automatic measurement of the tissue bridges using the proposed method on manually segmented (GT) intramedullary lesions (located under `derivatives/labels`)
+3. **_automatic_**: automatic measurement of tissue bridges using SCIsegV2 predictions
+
+## 0. Generate QC
+
+Generate QC to compare ground truth vs predicted lesions.
+
+### Ground truth
+```bash
+cd ~/data/data.neuro.polymtl.ca/sci-zurich
+for sub in sub-zh1[0-9][0-9];do sct_qc -i ${sub}/ses-01/anat/${sub}_ses-01_acq-sag_T2w.nii.gz -d derivatives/labels/${sub}/ses-01/anat/${sub}_ses-01_acq-sag_T2w_lesion-manual.nii.gz -s derivatives/labels/${sub}/ses-01/anat/${sub}_ses-01_acq-sag_T2w_seg-manual.nii.gz -p sct_deepseg_lesion -plane sagittal -qc-subject $sub -qc ~/data/results/sci-zurich/sci-zurich_2024-06-18_manualGT/qc;done
+```
+
```

Comment:
Add a note saying that the manual measurements were only provided for subjects `sub-zh101` to `sub-zh119`

Reply:
good point! added in https://github.com/ivadomed/model_seg_sci/pull/88/commits/e2780a04a99c647bca417851de52fe98ac5bd91c